### ISSUE TITLE
DOOM VFR (650000)

### ISSUE THREAD

**Issue by Shotweb** [ValveSoftware/Proton#5245]
**Describe the bug**
DOOM VFR crashes on launch. Tried with Proton Experimental, 6.x and 5.x.

- Distribution: pop_os 20.04, both using Gnome and Lxqt
- SteamVR version: 1.19.7 and beta 1.20.4
- Steam client version: 13. october 2021
- Opted into Steam client beta?: no
- Graphics driver version:  Mesa 21.2.1
- Gist for SteamVR System Information: [https://gist.github.com/Shotweb/607072b8a9ef51f122e34367336400e6](https://gist.github.com/Shotweb/607072b8a9ef51f122e34367336400e6)
- Console output: [https://gist.github.com/Shotweb/3f8bfaa6bd4a20af2d5e9574faddf010](https://gist.github.com/Shotweb/6c41b878f584408c1e8f4d9ba85f6591)

**Comment by kisak-valve** [comment]
Hello @Shotweb, issues with games run with Proton are being tracked in the Proton issue tracker, so I've gone ahead and transferred this issue report.

Please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by Shotweb** [comment]
Thank you,

I can add that I very briefly see a small and seemingly empty window appear and immediately disappear, when trying to launch the game.

[steam-650000.log](https://github.com/ValveSoftware/Proton/files/7403534/steam-650000.log)

Steam system information [https://gist.github.com/Shotweb/214ee1edaeebc88012fac0cf8b477096](https://gist.github.com/Shotweb/214ee1edaeebc88012fac0cf8b477096)

**Comment by Zamundaaa** [comment]
Same problem. [steam-650000.log](https://github.com/ValveSoftware/Proton/files/7540300/steam-650000.log), [system info](https://gist.github.com/Zamundaaa/1b1a7a6ce5d703b50843618ac59abac0)

**Comment by douggle** [comment]
So this game is still unable to start up in 2026 this is under proton 10

[steam-650000.log](https://github.com/user-attachments/files/24983721/steam-650000.log)

[systeminformation.txt](https://github.com/user-attachments/files/24983728/systeminformation.txt)

**Comment by kisak-valve** [comment]
Hello @douggle, these look like some lines of interest from your Proton log:
```
Error [GENERAL | xrCreateInstance | OpenXR-Loader] : LoaderInstance::CreateInstance chained CreateInstance call failed
Error [GENERAL | xrCreateInstance | OpenXR-Loader] : xrCreateInstance failed
```

**Comment by douggle** [comment]
Yeah I m wondering why,  my vr headset works with other titles with OpenXR as well ...

**Comment by douggle** [comment]
I just noticed this log was added in the Documents folder. 

[ErrorLog_01-31-2026__08-27-51am.txt](https://github.com/user-attachments/files/24983893/ErrorLog_01-31-2026__08-27-51am.txt)
